SimTracker/SiPixelDigitizer/plugins/SiPixelDigitizerAlgorithm.h

```diff
@@ -268,7 +280,9 @@ class SiPixelDigitizerAlgorithm  {
      enum shiftEnumerator {FPixRocIdShift = 3, BPixRocIdShift = 6};     
      static const int rocIdMaskBits = 0x1F;      
      void init_from_db(const edm::ESHandle<TrackerGeometry>&, const edm::ESHandle<SiPixelDynamicInefficiency>&);
-     bool matches(const DetId&, const DetId&, const std::vector<uint32_t >&);
+     bool matches(const DetId&, const DetId&, const std::vector<uint32_t >&);     
+     std::unique_ptr<PixelFEDChannelCollection> PixelFEDChannelCollection_;
```

Comment:
As the PixelFEDChannelCollection needs to be output to the event, the std::unique_ptr<PixelFEDChannelCollection> is returned by SiPixelFEDChannelContainer

Reply:
`SiPixelFEDChannelContainer` could return just `PixelFEDChannelCollection` (hopefully eventually by const reference), in which case it could be wrapped to `unique_ptr` only for the event.